ding/ding.py

```diff
@@ -120,16 +123,24 @@ def beep(seconds, command):
 
 def parse_time(args):
     """Figure out the number of seconds to wait"""
-    relative = args.mode == 'in'
+    relative = args.mode == 'in' or args.mode == "every"
+    #print(args.time)
     parser = TimeParser(args.time, relative)
     return parser.get_seconds()
 
 
 def main(args=sys.argv[1:]):
     args = get_args(args)
-    seconds = parse_time(args)
-    countdown(seconds, args.no_timer)
-    beep(seconds, args.command)
+    once = True
+    while once or args.mode == "every":
```

Comment:
Maybe we could just use `while True:`, and then at the end of the loop
```python
if not args.every: break
```
I'm not sure if this is necessarily better but it eliminates the need of the `once` variable. Best would be if there was a `do...while` in Python :stuck_out_tongue: